WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -209,6 +210,14 @@ public void onCreate() {
             }
         }
 
+        // Migrate self-hosted sites to FluxC
+        List<SiteModel> siteList = WPLegacyMigrationUtils.migrateSelfHostedSitesFromDeprecatedDB(this, mDispatcher);
```

Comment:
We should not call that everytime the app run, can we add an AppPref so we can check if the sites have already been migrated? 

Looking at our analytics, I see that old versions of the app are still in use, so I think we'll keep this migration process for 6 months or so, would be great to avoid any latency during app startup for users who don't need it.